spec/lib/cap_authors_poller_spec.rb

```diff
@@ -82,14 +81,12 @@
     context 'with an existing author' do
       before do
         expect(Author).to receive(:find_by_cap_profile_id).with(author.cap_profile_id).and_return(author)
-        expect(author).to receive(:'save!').and_call_original
+        expect(author).to receive(:'save!')
       end
 
       it 'updates an existing author' do
         expect(author).to receive(:harvestable?).and_return(true)
-        count = subject.instance_variable_get('@authors_updated_count')
-        subject.process_record(author_record)
-        expect(subject.instance_variable_get('@authors_updated_count')).to eq(count + 1)
+        expect { subject.process_record(author_record) }.to change { subject.instance_variable_get('@authors_updated_count') }.by(1)
       end
```

Comment:
:+1: for the revised syntax.

Long lines are difficult to review on github; pls consider breaking them for easier reading/review, e.g. any variant where it can break the line will do and this is just one option:
```ruby
expect { subject.process_record(author_record) }
  .to change { subject.instance_variable_get('@authors_updated_count') }.by(1)
```